transactions/btc.ts

```diff
@@ -66,18 +66,9 @@ export function selectUnspentOutputs(
     sumValue += pinnedOutput.value;
   }
 
-  // Sort UTXOs based on block time in ascending order
-  unspentOutputs.sort((a, b) => {
-    if (a.status.block_time && b.status.block_time) {
-      return a.status.block_time - b.status.block_time;
-    } else if (a.status.block_time) {
-      return 1;
-    } else if (b.status.block_time) {
-      return -1;
-    } else {
-      return a.value - b.value;
-    }
-  });
+  // Sort UTXOs based on value from largest to smallest
+  // This will give close to the optimal spend of UTXOs to minimise fees
+  unspentOutputs.sort((a, b) => b.value - a.value);
```

Comment:
Note this change. It will select UTXOs for transactions in a way to minimise fees better. Hence all the changes in the tests.

Reply:
sorting by confirmation time allowed us to have a chance of `CPFP` to speed up transactions any chance we can do both here?

Reply:
Ah, didn't realise that. Will add it back if the values are the same.